docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,104 @@
+| Show changelogs                           | Yes                                                                                                                                                                          | Yes                                                                                                      |
+| Compatibility score badges                | 1                                                                                                                                                                            | 4                                                                                                        |
+| Built-in to GitHub                        | Yes                                                                                                                                                                          | No, requires app or self-hosting                                                                         |
+| Scheduling                                | Yes: `daily`, `weekly`, `monthly`                                                                                                                                            | Yes: custom                                                                                              |
+| License                                   | [The Prosperity Public License 2.0.0](https://github.com/dependabot/dependabot-core/blob/main/LICENSE)                                                                       | [GNU Affero General Public License](https://github.com/renovatebot/renovate/blob/main/license)           |
+
+## Dependency Dashboard
+
+One big feature of Renovate is the Dependency Dashboard, which is enabled by default.
+Read the [Key concepts, Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) page to learn more.
+
+Dependabot does not have a similar feature.
+
+## Grouped updates
+
+Renovate comes with community-provided groupings of dependencies, so it will group common dependencies out-of-the-box.
+
+With Dependabot you'll have to set your own [`groups`](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups).
+
+## Upgrades common monorepo packages at once
+
+Renovate has a [`group:monorepos`](https://docs.renovatebot.com/presets-group/#groupmonorepos) preset, that upgrades common monorepo packages at once.
+
+Dependabot does not update common monorepo packages at once.
+
+## Supported platforms
+
+Platform means the Git-hosting site or program, for example GitHub, GitLab or Azure.
+
+Dependabot only works on GitHub.
```

Comment:
This is only true if compared with native `GitHub` integration.

The open-source component [dependabot-core](https://github.com/dependabot/dependabot-core) does allow creating solutions for other platforms.

* currently broken but I believe it used to support gitlab, bitbucket and probably other platforms: https://github.com/dependabot/dependabot-script
* community maintained (mostly by me) version that works with GitLab: https://gitlab.com/dependabot-gitlab/dependabot

Disclaimer, I'm not a `dependabot` maintainer or otherwise associated with `dependabot` team or `GitHub`

Reply:
Any comparison should indeed mention these, although be as impartially correct as possible about the others not being _supported_ in any official way. The first repo being broken for 6+ months kind of speaks for itself

Reply:
For sure, can't disagree here, but still, `Dependabot only works on GitHub.` is somewhat incorrect, maybe addition that `officially supported part only works on GitHub` or somewhere along those lines.

I imagine it's in the similar realm as `Renovate comes with community-provided groupings of dependencies, so it will group common dependencies out-of-the-box.` though I do imagine that `renovate` team is probably a bit more involved in supporting those community provided groupings 😄

Reply:
I think that the distinction here should highlight that the Dependabot BOT (not core) only works in Github. The average user won't be able to setup dependabot-core and create a bot using it. It is possible(I have done it) but it takes a considerable effort to do so. 

In that direction, one other point is that renovate is fully open source while the full dependabot BOT is not. Only the core framework is open and available to be used for further customizations  as @andrcuns mentioned,

Reply:
```suggestion
The _official_ Dependabot program only works on GitHub.

If you're an advanced user, you may use the [`dependabot-core` repository](https://github.com/dependabot/dependabot-core) as a base to build your own Dependabot, which you can run on other platforms.
```